### DISCUSSION TITLE
Tweaking speed threshold for a slow job

### DISCUSSION THREAD

**Discussion by reallyuniquename** [sabnzbd/sabnzbd#2164] [Q&A] [answered]
How do I tweak what SABnzbd considers a "slow job"? I'm talking about `Only get articles for top of the queue` option. Say, I want everything below 1MB/s to be deemed as a slow job but everything above that should actually block the queue.
Couldn't find anything in the docs.

**Comment by Safihre** [comment, marked as answer]
There's no such thing. We don't know what a slow job is, there's no per job speed.
I would recommend having only a few servers active (2 or 3), so trying all servers doesn't take long. If you haven't changed too many settings from their default, SABnzbd can then stop the job pretty quickly.

**Reply by reallyuniquename** [reply]
How does SABnzbd decide which job is a slow one?

**Reply by Safihre** [reply]
It doesn't. It checks all articles on all your servers. If there are more articles missing then there are par2 files that could be used for repair, the job is aborted.
Additionally, we also download the first article of each file. If alle files have their first articles missing, we also abort. This is the fastest way for fully removed jobs.

The Top of queue option does exactly as it says, it blocks severs from looking for articles for the second job in the queue. If a server has tried all articles for the top job, it can already try working on the second one.

**Reply by reallyuniquename** [reply]
So when SABnzbd finishes looking up all articles for the first job on server1 and server2 is too slow to keep up then second job gets to use server1 simultaneously. Is that correct?

**Reply by Safihre** [reply]
Exactly.
